contracts/dHedge-Factory-Version/Libraries/dHedgeHelper.sol

```diff
@@ -158,6 +158,7 @@ library dHedgeHelper {
             (, int96 _flowRate) = tokenData.superToken.getFlow(_sender);
             uint256 _currFlowRate = uint256(uint96(_flowRate));
             
+            // Not really necessary ?
             assert(
```

Comment:
I think is a good practice to check invariances. I would leave it.

Reply:
Check the if else block : _depositAmount > _userUninvested. What should happen when _depositAmount = _userUninvested?

Reply:
Ideally nothing should happen as _depositAmount == _userUninvested is only possible if a user updates a stream with the same stream rate as before updation. For example if my current stream rate is $10/hr and I send a transaction to update this to $10/hr then nothing should happen.